[2025-01-05T13:52:54.688+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-05T13:52:54.888+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_spark_cluster.spark_action manual__2025-01-05T13:52:49.578995+00:00 [queued]>
[2025-01-05T13:52:55.025+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_spark_cluster.spark_action manual__2025-01-05T13:52:49.578995+00:00 [queued]>
[2025-01-05T13:52:55.035+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-05T13:52:55.254+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): spark_action> on 2025-01-05 13:52:49.578995+00:00
[2025-01-05T13:52:55.309+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=96) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-05T13:52:55.334+0000] {standard_task_runner.py:72} INFO - Started process 119 to run task
[2025-01-05T13:52:55.343+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_spark_cluster', 'spark_action', 'manual__2025-01-05T13:52:49.578995+00:00', '--job-id', '566', '--raw', '--subdir', 'DAGS_FOLDER/dag_spark_cluster.py', '--cfg-path', '/tmp/tmp3daulswk']
[2025-01-05T13:52:55.394+0000] {standard_task_runner.py:105} INFO - Job 566: Subtask spark_action
[2025-01-05T13:52:56.036+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_spark_cluster.spark_action manual__2025-01-05T13:52:49.578995+00:00 [running]> on host 44275986e897
[2025-01-05T13:52:56.455+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Anoop M' AIRFLOW_CTX_DAG_ID='dag_spark_cluster' AIRFLOW_CTX_TASK_ID='spark_action' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T13:52:49.578995+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-05T13:52:49.578995+00:00'
[2025-01-05T13:52:56.473+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-05T13:52:57.716+0000] {logging_mixin.py:190} INFO - Error starting Spark Master: 409 Client Error for http+docker://localhost/v1.47/containers/create?name=spark-master: Conflict ("Conflict. The container name "/spark-master" is already in use by container "a1d84de1c089b00491ad061b353e8a7548b3e53462ef1816ff161c8781b8c0be". You have to remove (or rename) that container to be able to reuse that name.")
[2025-01-05T13:52:59.156+0000] {logging_mixin.py:190} INFO - Error starting Spark Worker: 409 Client Error for http+docker://localhost/v1.47/containers/create?name=spark-worker: Conflict ("Conflict. The container name "/spark-worker" is already in use by container "fc2ac179aef4e250cc3c7ec3ad16e384dd081c040bb2464a91cc0d625170604b". You have to remove (or rename) that container to be able to reuse that name.")
[2025-01-05T13:52:59.226+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-05T13:52:59.724+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-05T13:52:59.763+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_spark_cluster, task_id=spark_action, run_id=manual__2025-01-05T13:52:49.578995+00:00, execution_date=20250105T135249, start_date=20250105T135254, end_date=20250105T135259
[2025-01-05T13:53:01.281+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-05T13:53:01.461+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
